share/unit/tests/core.sh

```diff
@@ -57,6 +57,12 @@ function testCoreGlobal() {
     #assertEquals "1.4" 2600 $v
 }
 
+function testCoreUnsupportedAssociativeArrayAssignments() {
+    local vetted
+    vetted="$(git grep -E '[a-zA-Z0-9]+\+=\( *\[' | md5sum | cut -d' ' -f1)"
+    assertEquals '0.0.1' '09a2684a0023bdd670ad455efbd74d8e' "${vetted}"
```

Comment:
${vetted%% *} would avoid the last cut and one process but we would lose code readability. So I actually tend to reject my own comment since the performance impact is neglectable.